README.md

```diff
@@ -37,6 +37,71 @@ Latest API Docs is <a target="_blank" href="https://wso2-extensions.github.io/si
      </dependency>
 ```
 
+## Running Integration tests in docker containers(Optional)
+
+1. Install and run docker in daemon mode.
+
+    *  How to install docker on Linux
```

Comment:
The command given below should be preceded by a sentence that specifies why it is issued and from which location it is issued.

e.g., To install and run Docker in 'daemon` mode, navigate to the <RELEVANT_DIRECTORY_PATH> and issue the following command.

Reply:
There is no specific location for issues this command, these are the bash script commands which use for install the docker. Yes, we can specify the purpose.